roles/container_runtime/templates/crio.conf.j2

```diff
@@ -27,7 +27,11 @@ storage_option = [
 [crio.api]
 
 # listen is the path to the AF_LOCAL socket on which crio will listen.
+{% if openshift.common.version_gte_3_9 %}
```

Comment:
We should be using crio's install version to determine this, not openshift.  Whatever we're setting to ensure we get the right version of crio here (an image tag probably), that's what we need to check.

Reply:
@runcom is the install for the ``cri-o`` container still using ``latest``? If so, that would be the first area to "fix" so the version could be sensed from the tag.

Reply:
installer use 3.9/3.8/3.7 now

Reply:
Ok. Then the change that needs to happen is to use the ``version_compare`` filter as @michaelgugino or update the facts module to default to 3.9 instead of 3.8 in master. Either of those should get you past the problem.

Reply:
@runcom knows better, but I think this really depend on the version of openshift/k8s, the same changee goes in `node.yaml.v1.j2`

Reply:
Either way *should* still end up depending on the version of OpenShift (tag version matches Openshift version) ... but I'll leave what path to take up to you guys.